scripts/rff/climate_wildfire_smoke/ClimateWildfireSmoke.tmcf

```diff
@@ -0,0 +1,199 @@
+Node: E:ClimateWildfireSmoke->E0

```

Comment:
Could you replace this file with the content in this one:
[ClimateWildfireSmoke.tmcf.txt](https://github.com/datacommonsorg/data/files/11205771/ClimateWildfireSmoke.tmcf.txt)

(This has been updated to use the checked-in schema in this commit: https://github.com/datacommonsorg/schema/commit/374e2becb7e87a7f6121105791ac2cda0d422a35)

Reply:
Changes are reflected in the new commits